set.mm

```diff
@@ -162305,6 +162363,13 @@ the complex numbers (sorting by real part first, then by imaginary part as
       BCKZHABLMNBOPCLMNCOPUAUBQDEFGBCRST $.
   $}
 
+  $( The absolute value of the imaginary component of the square root of a
+     complex number is a real number.  (Contributed by RP, 11-May-2024.) $)
```

Comment:
wrong comment

Reply:
Abs[Im[Sqrt[z]]] = Sqrt[(Abs[z] - Re[z])/2] holds for all z in the complex plane and is a real number.